spec/index.bs

```diff
@@ -8075,6 +8077,14 @@ dictionary GPUFragmentState
                     : {{GPUTextureSampleType/"uint"}}
                     :: |output| must have an unsigned integer scalar type.
                 </dl>
+            - If |colorState|.{{GPUColorTargetState/blend}}.{{GPUBlendState/color}}.{{GPUBlendComponent/srcFactor}} or
+                 |colorState|.{{GPUColorTargetState/blend}}.{{GPUBlendState/color}}.{{GPUBlendComponent/dstFactor}} or
+                 |colorState|.{{GPUColorTargetState/blend}}.{{GPUBlendState/alpha}}.{{GPUBlendComponent/srcFactor}} or
+                 |colorState|.{{GPUColorTargetState/blend}}.{{GPUBlendState/alpha}}.{{GPUBlendComponent/dstFactor}}
+                 uses src1 (is any of {{GPUBlendFactor/"src1"}}, {{GPUBlendFactor/"one-minus-src1"}},
+                 {{GPUBlendFactor/"src1-alpha"}}, {{GPUBlendFactor/"one-minus-src1-alpha"}}):
+                 - |output| must have a [=blend_src=] attribute.
+                 - |descriptor|.{{GPUFragmentState/targets}}.length must be `1`.
             - If |colorState|.{{GPUColorTargetState/blend}} is [=map/exist|provided=] and
                 |colorState|.{{GPUColorTargetState/blend}}.{{GPUBlendState/color}}.{{GPUBlendComponent/srcFactor}}
                 or .{{GPUBlendComponent/dstFactor}} uses the source alpha
```

Comment:
The enum values specified in this rule should differ depending on whether we're on src or src1.

Reply:
I've updated the statements here. Could you take a look?